docs/dataset/formatting-your-datasets.md

```diff
@@ -23,26 +23,34 @@ dataset with the following columns:
 From this you can see that image `horse0000.png` has the ground_truth classification of `horse`,
 and has brightness and contrast data.
 
-An `id_field` is required in order to differentiate between datapoints.
-By default the `locator` or  `text` fields are used if present in your dataset, and other fields
-can be specified when importing via the Web App from the [:kolena-dataset-16: Datasets](https://app.kolena.io/redirect/datasets)
-page, or the SDK by using [`upload_dataset`](../reference/dataset/index.md#kolena.dataset.dataset.upload_dataset)
-function.
+When uploading a dataset to Kolena, it is important to be able to differentiate between each datapoint. This is
+accopmplished by configuring an `id_field` - an unique identifier for a datapoint. You can select any field that is
+unique across your data, for example a `uuid` or `article_id` for relevant data.
+
+Kolena handles the following `id_fields` as a special case:
+
+| Field Name  | Description                                                                                                                              |
+|-------------|------------------------------------------------------------------------------------------------------------------------------------------|
+| `locator`     | Url path to a file to be displayed, either a [cloud storage](../connecting-cloud-storage/index.md) url or a http url that serves a file. |
+| `text`        | Raw text input for text based models.                                                                                                    |
```

Comment:
These aren't necessarily `id_fields`. Can this be rewritten to something like "Kolena will look for the following fields when displaying datapoints"

Reply:
nit: can we keep tables aligned

Reply:
updated